cernopendata/modules/fixtures/data/records/cms-validation-code-Castor2010.json

```diff
@@ -0,0 +1,78 @@
+[
+    {
+      "abstract": {
+        "description": " <p>This record contains code that needs to be used to properly read out CASTOR Open Data of Commissioning10, Run2010A, and Run2010B datasets. It includes two EDAnalyzer plugins that are created to correctly read the reconstructed CASTOR data objects and make validation plots.</p>"
+      },
+      "accelerator": "CERN-LHC",
+      "authors": [
+        {
+          "name": "Van Haevermaet, Hans"
+        }
+      ],
+      "collections": [
+        "CMS-Validation-Utilities"
+      ],
+      "date_created": [
+        "2010"
+      ],
+      "date_published": "2020",
+      "distribution": {
+        "formats": [
+          "gz"
+        ]
+      },
+      "doi": "FIXME",
+      "experiment": "CMS",
+      "license": {
+        "attribution": "GNU General Public License (GPL) version 3"
+      },
+      "publisher": "CERN Open Data Portal",
+      "recid": "465",
+      "run_period": [
+        "Commissioning2010",
+        "Run2010A",
+        "Run2010B"
+      ],
+      "source_code_repository": {
+        "url": "https://github.com/cms-opendata-validation/CastorDataValidation"
```

Comment:
The repo has not been moved yet to `cms-opendata-validation/CastorDataValidation` so I guess we'll leave this on hold for a bit more still?

Reply:
Yes, I'll push the needed container in dockerhub and modify the container in the workflow and then it's good to go

Reply:
It is now in place and a release tag done https://github.com/cms-opendata-validation/CastorDataValidation/releases/tag/v1.0

Reply:
Reserved DOI is `10.7483/OPENDATA.CMS.ESAE.3S38`, to be registered when we move this to production.